docs/source/metabase.py

```diff
@@ -134,6 +134,14 @@ def _read_session_data(self) -> Optional[Dict]:
 
         return entry.get('data')
 
+    @staticmethod
+    def _none_if_blank(value: str) -> Union[str, None]:
+        """Return the value with whitespace stripped or None if it's blank."""
```

Comment:
Suggestions welcome on this name. The name may be horrible regardless, but the awkward part to me is that it's returning the stripped value rather than just using that to determine if the value is blank.

Reply:
I think this is fine, it's just a name. But here is an alternative with a more concise type, implementation, and slightly different name that directly mirrors the implementation:

```python3
@staticmethod
def stripped_or_none(value: str) -> Optional[str]:
    """Return the value with whitespace stripped or None if it's blank."""
    return value.strip() or None
```

Reply:
That is much nicer. Maybe later.

Is `Optional` correct? I had that originally, but I thought it was more intended for arguments. However, it does just appear to be an alias for `Union[T, None]`.

Reply:
I think it's just a bad name for that type.

Apparently these days you can spell Union[T, U] as `T | U` so this could become:

```python3
@staticmethod
def stripped_or_none(value: str) -> str | None:
    """Return the value with whitespace stripped or None if it's blank."""
    return value.strip() or None
```